core/server/data/schema/schema.js

```diff
@@ -205,8 +205,18 @@ module.exports = {
             defaultTo: 'public',
             validations: {isIn: [['public', 'internal']]}
         },
+        og_image: {type: 'string', maxlength: 2000, nullable: true},
+        og_title: {type: 'string', maxlength: 300, nullable: true},
+        og_description: {type: 'string', maxlength: 500, nullable: true},
+        twitter_image: {type: 'string', maxlength: 2000, nullable: true},
+        twitter_title: {type: 'string', maxlength: 300, nullable: true},
+        twitter_description: {type: 'string', maxlength: 500, nullable: true},
         meta_title: {type: 'string', maxlength: 2000, nullable: true, validations: {isLength: {max: 300}}},
         meta_description: {type: 'string', maxlength: 2000, nullable: true, validations: {isLength: {max: 500}}},
+        codeinjection_head: {type: 'text', maxlength: 65535, nullable: true},
+        codeinjection_foot: {type: 'text', maxlength: 65535, nullable: true},
+        canonical_url: {type: 'text', maxlength: 2000, nullable: true},
```

Comment:
@allouis I'm assuming the type is `text` here because there's an issue with `tags` table size? My understanding the conclusion for the discussion around type here was to go with `type: 'string'` unless it's the same situation as `posts` table's :)